apps/ios/src/Podfile

```diff
@@ -29,6 +29,7 @@ use_test_app! do |target|
     pod 'FluentUI-React-Native-Date-Picker', :path => '../../../packages/experimental/NativeDatePicker/FluentUIReactNativeDatePicker.podspec'
     pod 'react-native-menu', :path => '../../../node_modules/@react-native-menu/menu'
     pod 'RNSVG', :path => '../../../node_modules/react-native-svg'
+    pod 'react-native-slider', :path => '../../../node_modules/@react-native-community/slider'
```

Comment:
Thanks! I never actually told you how to do this 😅 sorry for not following up!

Reply:
I just had a quick look at the Podfile, and I said to myself lets try it! And it worked, so happy =)